commit.log

```diff
@@ -2126,7 +2126,7 @@ Date:   Mon Oct 28 16:26:50 2013 -0700
     Differentiate between normal ternary (a ? b : c) and short ternary, (b ?: c) with option sp_cond_ternary_short
 
 commit 4b83d06c86ab949d1bc71d8eaa2cee9e9c7332bb
-Author: Tim Hütz <tim@huetz.biz>
+Author: Time Hütz <time@huetz.biz>
```

Comment:
I think this fix is incorrect.

Reply:
Changed.

Reply:
The email is now ok but the same still says `Time Hütz`, which is wrong

Reply:
Sorry, corrected that as well.